src/main/resources/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealm/config.jelly

```diff
@@ -33,6 +33,9 @@ THE SOFTWARE.
     <f:textbox field="headerGroupsDelimiter" default="|" />
   </f:entry>
   <f:advanced>  
+    <f:entry title="${%Custom Log In URL}" >
+      <f:textbox name="customLogInUrl" value="${instance.customLogInUrl}" />
```

Comment:
This should have an associated help file as this doesn't seem like a feature with obvious behavior.